core/domain/exp_domain.py

```diff
@@ -3968,6 +4008,19 @@ def get_interaction_ids(self):
             state.interaction.id for state in self.states.itervalues()
             if state.interaction.id is not None]))
 
+    def get_all_html_content_strings(self):
+        """Gets all html content strings used in this exploration.
+
+        Returns:
+            list(str). The list of html content strings.
+        """
+        html_list = []
+        for state in self.states.itervalues():
+            content_html = state.content.html
+            interaction_html = state.interaction.get_all_html_content_strings()
```

Comment:
Naming is not so good -- this is a list, the previous line is a string. Call this interaction_html_list instead.

Reply:
Done!